Util/Xamarin.AndroidBinderator/Xamarin.AndroidBinderator/Engine.cs

```diff
@@ -326,6 +326,15 @@ static List<BindingProjectModel> BuildProjectModels(BindingConfig config, Dictio
 
 					mavenDep.Version = FixVersion(mavenDep.Version, mavenProject);
 
+					if (mavenDep.GroupId.Contains("${project.groupId}"))
```

Comment:
You don't need to do the `Contains` here if you don't want to, you can just call `Replace` and if nothing is found it won't change the string.

Reply:
fixed